avocado/core/loader.py

```diff
@@ -717,7 +719,8 @@ def _make_avocado_tests(self, test_path, make_broken, subtests_filter,
                 else:
                     # Module does not have an avocado test class inside, and
                     # it's not executable. Not a Test.
-                    return make_broken(NotATest, test_path)
+                    return make_broken(NotATest,
```

Comment:
This works, but as a suggestion, `make_broken()` could take a message parameter and do the formatting itself.  It would ensure that future code would print messages in the proper format.